### PR TITLE
fix "Authorization: Bearer Undefined" on Swagger

### PR DESCRIPTION
**Pull request by nullhack**
This PR solves the issue https://github.com/frankie567/fastapi-users/issues/212

**Problem**

After authentication on Swagger documentation, the button `Try it out` generates `Undefined` instead of filling the Token of authenticated user.

**Solution**

Change the return of `get_login_response` on [jwt.py](https://github.com/frankie567/fastapi-users/blob/master/fastapi_users/authentication/jwt.py#L68) to match the expected return described at [fastapi's guides](https://github.com/tiangolo/fastapi/blob/master/docs_src/security/tutorial004.py#L128)

**Testing**

image

### CODE REVIEW

**Comment by frankie567** [comment]
Very nice catch, @nullhack! Thank you very much 👍

File: tests/test_authentication_jwt.py [hunk 1]
```diff
@@ -68,9 +68,9 @@ async def test_valid_token(self, jwt_authentication, mock_user_db, token, user):
 async def test_get_login_response(jwt_authentication, user):
     login_response = await jwt_authentication.get_login_response(user, Response())
 
-    assert "token" in login_response
+    assert "access_token" in login_response
```
Comment (frankie567) [inline tests/test_authentication_jwt.py:71]: We should add `assert login_response["token_type"] == "bearer"` here

Comment (nullhack) [reply tests/test_authentication_jwt.py:71]: Sure, I'll do during the weekend

**Comment by martincolladofab** [comment]
What action is necessary to merge and finish this PR?

**Comment by nullhack** [comment]
@martincolladofab This is done. Just need a review and merge.

**Comment by frankie567** [comment]
Good to go! Thank you very much @nullhack!

**Pull request merged**